catalyst_voices/lib/widgets/containers/roles_chooser_panel.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:catalyst_voices/widgets/cards/role_chooser_card.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/widgets.dart';
+
+/// A panel that displays a series of [RoleChooserCard] widgets for selecting
+/// various account roles (e.g., voter, proposer, drep).
+class RolesChooserPanel extends StatelessWidget {
```

Comment:
`Panel` is smth we call side of registration dialog. Maybe just `RolesChooser`?

Reply:
Then, I will call both to `RolesChooserContainer` and `RolesSummaryContainer`, as they both act as a layout for containing roles, and the actual roles are defined here.

Reply:
Makes sense 👍